src/data/information-technology-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const informationTechnologyQuiz = [
+  {
+    Question: "How many layers are there in OSI(Open Systems Interconnection) and (TCP/IP)Transmission Control Protocol/Internet Protocol Models respectively",
```

Comment:
Long question usually don't look great on the site. Let's shorten this

```suggestion
    Question: "How many layers are there in the OSI and TCP/IP models respectively?",
```